evap/staff/tests/test_forms.py

```diff
@@ -430,3 +431,51 @@ def test_contributor_course_form_date_validation(self):
 
         # staff: but start date must be < end date
         self.helper_date_validation(CourseForm, "02/1/1999", "02/1/1998", False)
+
+
+class TestCourseOperationForms(WebTest):
+    fixtures = ['minimal_test_data']
+
+    def helper_semester_state_views(self, course_ids, old_state, new_state, operation):
```

Comment:
It does, and I think the best option would be to remove the tests in test_views as they are incomplete